recipe/meta.yaml

```diff
@@ -10,19 +10,21 @@ source:
 
 build:
   noarch: python
-  number: 3
```

Comment:
We possibly need to mark all previous builds as broken as they would otherwise be preferred by the solver due to less dependency constraints. As the link scripts are broken themselves, it is a good cause to mark them as broken.

Reply:
how do we do that?

Reply:
Make a PR to https://github.com/conda-forge/admin-requests/